paf-mvp-operator-express/src/operator-node.ts

```diff
@@ -765,14 +765,26 @@ export class OperatorNode extends Node {
 
   catchErrorsWithRedirectToReferer =
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    (error: unknown, req: Request, res: Response, next: NextFunction) => {
+    (err: unknown, req: Request, res: Response, next: NextFunction) => {
       const { endPointName } = this.getRequestConfig(req);
-      this.logger.Error(endPointName, error);
-      const nodeError: NodeError = {
-        type: NodeErrorType.UNKNOWN_ERROR,
-        details: '',
-      };
-      this.redirectWithError(res, req.header('referer'), 500, nodeError);
+      this.logger.Error(endPointName, err);
+
+      // FIXME[errors] this whole method must be merged with catchErrors to avoid duplication like this
```

Comment:
Have you considered a previous suggestion regarding an error handler for REST API and another one for the Redirect API? I believe that the code would be simpler and safer. If you concern is the code duplication, the composition of functions or queueing error handler can do the job.  Maybe to discuss with Amine.